example_app_generator/spec/verify_custom_renderers_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe "template rendering", :type => :controller do
+  context 'with the standard renderers' do
+    controller do
+      def index
+        render :template => 'foo'
+      end
+    end
+
+    it "renders the 'foo' template" do
+      get :index
+
+      expect(response).to render_template(:foo)
+    end
+  end
+
+  context 'with a String path prepended to the view path' do
+    controller do
+      def index
+        prepend_view_path('app/views/some_templates')
+
+        render :template => 'bar'
+      end
+    end
+
+    it "renders the 'bar' template" do
+      get :index
+
+      expect(response).to render_template(:bar)
+    end
+  end
+
+  context 'with a custom renderer prepended to the view path' do
+    controller do
+      def index
+        prepend_view_path(MyResolver.new)
+
+        render :template => 'baz'
+      end
+    end
+
+    it "renders the 'baz' template" do
+      get :index
+
+      expect(response).to render_template(:baz)
+    end
```

Comment:
Great start, we just need to check that the views are empty (as `render_views` hasn't been used) and ideally have a second set of tests that checks that the views can be rendered when specified. Appreciate your work to get this towards the line!